.changesets/2420.md

```
@@ -0,0 +1,20 @@
+- feat(cli): Add `skipWatchWarning` opt-out for `cedar dev` package-watch warning by @lisa-assistant
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)